### PR TITLE
Use sdk/api to get contract data

### PR DESCRIPTION
**Pull request by bthaile**
first consolidation of contract calls to get values. Didn't move anything tx still need changes in the middleware before continuing that effort.

summary:
remove `currentBlockTimestamp`
converted callbacks to async await

https://github.com/augurproject/augur/issues/1983

### CODE REVIEW

File: packages/augur-ui/src/modules/contracts/actions/contractCalls.ts [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+ * 2. pending tx and how to support existing in-line processing feedbadk to user
+ */
+// put all calls to contracts here that need conversion from display values to onChain values
+import { augurSdk } from "services/augursdk";
+import { ethers } from "ethers";
+import { BigNumber } from "ethers/utils";
+
+export async function checkIsKnownUniverse(universeId: string) {
+  const { contracts } = augurSdk.get();
+  const result = await contracts.augur.isKnownUniverse_(universeId);
```
Comment (justinbarry) [inline packages/augur-ui/src/modules/contracts/actions/contractCalls.ts]:
```suggestion
return await contracts.augur.isKnownUniverse_(universeId);
```

File: packages/augur-ui/src/modules/universe/actions/load-universe-info.ts [hunk 2]
```diff
@@ -251,6 +259,22 @@ function getUniverseName(parentUniverseData: any, universeData: any) {
...
+    const isForking =
+      forkingMarket !== "0x0000000000000000000000000000000000000000";
+    if (isForking) {
+      const forkEndTime = await getForkEndTime();
+      const forkReputationGoal = await getForkReputationGoal();
+      const isForkingMarketFinalized = await isFinalized(forkingMarket);
+      let winningChildUniverse;
+      if (isForkingMarketFinalized) {
+        winningChildUniverse = await getWinningChildUniverse();
+=======
```
Comment (JohnDanz) [inline packages/augur-ui/src/modules/universe/actions/load-universe-info.ts]: missed a merge conflict the `<<<<<< HEAD:` and `=======`

File: packages/augur-ui/src/modules/universe/actions/load-universe-info.ts [hunk 3]
```diff
@@ -327,11 +351,20 @@ export function getForkingInfo(universe: any, callback: Function = logError) {
             winningChildUniverse: undefined
           });
         }
+>>>>>>> master:packages/augur-ui/src/modules/universe/actions/load-universe-info.ts
```
Comment (JohnDanz) [inline packages/augur-ui/src/modules/universe/actions/load-universe-info.ts]: Missed a merge conflict

File: packages/augur-ui/src/modules/universe/actions/load-universe-info.ts [hunk 4]
```diff
@@ -351,6 +384,14 @@ function updateUniverseIfForkingDataChanged(
 }
 
 export function getUniverseProperties(universe: any, callback: Function) {
+<<<<<<< HEAD:packages/augur-ui/src/modules/universe/actions/load-universe-info.ts
```
Comment (JohnDanz) [inline packages/augur-ui/src/modules/universe/actions/load-universe-info.ts]: Missed merge conflict

File: packages/augur-ui/src/modules/universe/actions/load-universe-info.ts [hunk 5]
```diff
@@ -386,5 +427,6 @@ export function getUniverseProperties(universe: any, callback: Function) {
         if (callback) callback(err, universeData);
       }
     );
+>>>>>>> master:packages/augur-ui/src/modules/universe/actions/load-universe-info.ts
```
Comment (JohnDanz) [inline packages/augur-ui/src/modules/universe/actions/load-universe-info.ts]: missed merge conflict

File: packages/augur-ui/src/services/augurjs.js [hunk 6]
```diff
@@ -1,25 +1,25 @@
 import Augur from "@augurproject/augur.js";
```
Comment (JohnDanz) [inline packages/augur-ui/src/services/augurjs.js:1]: should prob convert this file to `augurjs.ts`

**Review by JohnDanz — CHANGES_REQUESTED** [review]
couple things to cleanup

File: packages/augur-ui/src/services/augurjs.js (hunk 6 above)
Comment (bthaile) [reply packages/augur-ui/src/services/augurjs.js:1]: converted to .ts file

**Review by JohnDanz — APPROVED** [review]

**Pull request merged**